l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -105,7 +128,7 @@ contract AdminFacet is Base, IAdmin {
 
     /// @notice Instantly pause the functionality of all freezable facets & their selectors
     /// @dev Only the governance mechanism may freeze Diamond Proxy
-    function freezeDiamond() external onlyGovernor {
+    function freezeDiamond() external onlyGovernorOrStateTransitionManager {
```

Comment:
Should we add a method to STM to call `freezeDiamond`? Otherwise I don't see how it can be called